src/dotnet/ToolManifest/LocalizableStrings.resx

```diff
@@ -150,4 +150,7 @@
   <data name="ManifestMissingIsRoot" xml:space="preserve">
     <value>Missing 'isRoot' entry.</value>
   </data>
+  <data name="ManifestPackageIdCollision" xml:space="preserve">
+    <value>Cannot add package {0} version {1} to manifest file {2}. Package {3} version {4} exists.</value>
```

Comment:
Cannot add package. Manifest file already contains version {0} of the package {}. Uninstall/install or edit the manifest file to change to version {}.